server/author/src/test/java/gov/ny/its/hs/maslow/author/controller/ServiceControllerTests.java

```diff
@@ -0,0 +1,284 @@
+package gov.ny.its.hs.maslow.author.controller;
```

Comment:
why do we have a new file? is this from simply rolling back the auth PR?

Reply:
yes the test file was removed when Okta was implemented, so i put it back